Userland/Libraries/LibWeb/Fetch/Fetching/Fetching.cpp

```diff
@@ -1901,11 +1901,12 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<PendingResponse>> http_network_or_cache_fet
         auto inner_pending_response = PendingResponse::create(vm, request, *response);
 
         // 14. If response’s status is 401, httpRequest’s response tainting is not "cors", includeCredentials is true,
-        //     and request’s window is an environment settings object, then:
+        //     request’s window is an environment settings object, and there is a WWW-Authenticate header, then:
```

Comment:
This comment is copied verbatim from the Fetch specification at https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch

If this step in the spec is incorrect, please file a bug against https://github.com/whatwg/fetch

If we *need* to do something differently here (because the spec is wrong), we can add a separate comment, something like:

```cpp
// FIXME: This isn't in the spec, but is actually required to avoid getting into a request loop.
//        Spec bug: https://github.com/whatwg/fetch/issues/...